docs/DeveloperGuide.md

```diff
@@ -1037,6 +1037,11 @@ Could not read data from file FILE_PATH due to inability to find or access the f
 This error message could be more specific and our team plans to update this to the following error message: <br>
 DorManagerPro could not access the file at FILE_PATH. This could be because the file is of the wrong type, it has a format incompatible with DorManagerPro or it contains invalid data. Please check that the file path leads to a JSON file with valid data and formatting.
 
+6. **Consistent behavior for `find`.** Currently, `find` are having inconsistent behavior on different fields which is counterintuitive. The problems are as following: <br>
```

Comment:
maybe can rephrase to:

"Currently, `find` has some inconsistent behaviour depending on the field that it is being searched by, making it counterintuitive for users."